ui/webui/resources/wasm/candle_embedding_gemma/.cargo/config.toml

```diff
@@ -0,0 +1,11 @@
+[source.crates-io]
+replace-with = "vendored-sources"
+
+[source.vendored-sources]
+directory = "../vendor"
+
+[target.wasm32-unknown-unknown]
+rustflags = [
+  '--cfg', 'getrandom_backend="wasm_js"',
+  '-C', 'target-feature=+simd128'
+]
```

Comment:
Just a heads-up: this will be wiped whenever we add a new dependency using `npm run update_wasm_resources`.

Reply:
this config.toml is specific in this PR so I did the change in this PR
98729a6fe87edf132029c9dfebf48cecba3e22d5